### PR TITLE
fix: Fixed overflowing username while adding team members

### LINKED ISSUE #11298: [CAL-2448] bug: create team UI doesnt handle overflow correctly
CleanShot 2023-09-12 at 11 27 57

<sub>[CAL-2448](https://linear.app/calcom/issue/CAL-2448/bug-create-team-ui-doesnt-handle-overflow-correctly)</sub>

### PR DESCRIPTION
**Pull request by pateldivyesh1323**
## What does this PR do?

Fixes #11298 

Fixed overflowing username if it is very long while adding team members

- After
trimmedusername

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

**Comment by Dilli-Raj-Timalsina** [comment]
Is there anybody working on this issue ? could anybody assign this issue to me ?

**Comment by PeerRich** [comment]
> Is there anybody working on this issue ? could anybody assign this issue to me ?

you are commenting on a PR, not an issue.

File: packages/features/ee/teams/components/AddNewTeamMembers.tsx [hunk 1]
```diff
@@ -240,9 +240,9 @@ const PendingMemberItem = (props: { member: TeamMember; index: number; teamId: n
         index !== 0 && "border-subtle border-t"
       )}
       data-testid="pending-member-item">
-      <div className="flex space-x-2 rtl:space-x-reverse">
+      <div className="flex w-[85%] space-x-2 rtl:space-x-reverse">
         <Avatar size="mdLg" imageSrc={bookerUrl + "/" + member.username + "/avatar.png"} alt="owner-avatar" />
-        <div>
+        <div className="w-[82%] sm:w-[95%]">
```
Comment (PeerRich) [inline packages/features/ee/teams/components/AddNewTeamMembers.tsx:245]: i wonder if this is possible without percentages

**Comment by PeerRich** [comment]
going with https://github.com/calcom/cal.com/pull/11320 which is cleaner with max-width

**Comment by PeerRich** [comment]
thank you tho!

**Pull request closed without merging**